packages/graphql-codegen-cli/src/codegen.ts

```diff
@@ -65,87 +68,211 @@ async function mergeSchemas(schemas: GraphQLSchema[]): Promise<GraphQLSchema> {
 }
 
 export async function executeCodegen(config: Types.Config): Promise<FileOutput[]> {
-  const result = [];
+  const result: FileOutput[] = [];
+  const commonListrOptions = {
+    exitOnError: true
+  };
+  const verboseOptions = {
+    ...commonListrOptions,
+    renderer: 'verbose',
+    nonTTYRenderer: 'verbose'
+  };
```

Comment:
In verbose mode, it won't show every frame, only `task started`, `task completed` etc.